server/controller/message/messages.js

```diff
@@ -105,6 +105,59 @@ class MessageController {
       next(e);
     }
   }
+
+  /**
+   * get all sent messages
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @param{function} next - next function
+   * @returns {object} response object
+   */
+  static async getMessage(req, res, next) {
+    const { userId } = req.user;
+    const { messageId } = req.params;
+    let message;
+
+    try {
+      const user = await User.findByPk(userId);
+      if (!user) {
+        return res.status(404).json({
+          status: 404,
+          error: 'This user does not exist'
+        });
+      }
+
+      const sentMessage = await Message.findOne({
+        where: { senderId: userId, id: messageId }
+      });
+
+      const receivedMessage = await Message.findOne({
+        where: { receiverId: userId, id: messageId }
+      });
+
+      if (!sentMessage && !receivedMessage) {
```

Comment:
What if there's a  `sentMessage` and no `receivedMessage`? 🤔 @beejay1293

Reply:
@fortune, that why I checked in the next line with this:
  if (!sentMessage && !receivedMessage) {
        return res.status(404).json({
          status: 404,
          message: 'no messages'
        });
      }

and this helps me to check which one contains a message to return to the user
 
if (sentMessage) {
        message = sentMessage.dataValues;
      } else {
        message = receivedMessage.dataValues;
      }

Reply:
Alright please resolve the conflicts @beejay1293

Reply:
Done @fortunee